src/link-manager.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @fileoverview Link manager for controlling the link query.
+ */
+
+/** @const */
+genotet.linkManager = {};
+
+/**
+ * Object that stores the link views by their type.
+ */
+/** @type {!Object<string, !Array<genotet.View>>} */
+genotet.linkManager.linkViews = {};
+
+/**
+ * Object that stores the link actions and responses type.
+ */
+/** @type {!Object<string>} */
+genotet.linkManager.linkActions = {};
```

Comment:
What action settings do you want to share? Action/response types can simply be enums.